gitgud/test_operator.py

```diff
@@ -22,6 +26,21 @@ def level():
     return all_skills["1"]["1"]
 
 
+@pytest.fixture
+def content_level(gg):
+    from gitgud.skills import all_skills
+    gg.load_level(all_skills['intro']['welcome'])
+
+    write_file("root.txt")
+    os.mkdir("dir")
+    write_file("dir/dirfile.txt")
+    os.mkdir("dir/subdir")
+    write_file("dir/subdir/subdirfile.txt")
```

Comment:
Why do you add `root.txt` and `subdirfile.txt`? It seems like you don't use them

Reply:
I've now added tests for get_commit_content using subdir*.